### PR TITLE
Update documentation for Az VM creation script

### PR DESCRIPTION
**Pull request by neerajdixit-msft**
This PR updates the README for VM creation.
This PR also changes some of the input arguments to the script to make it more user friendly

### CODE REVIEW

File: Scripts/AzPowerShellScript/AzHelper.psm1 [hunk 1]
```diff
@@ -196,13 +195,13 @@ function GetStorageAccount
     [CmdletBinding()]
     Param
     (
-        [Parameter(Mandatory=$false)]
+        [Parameter(Mandatory=$true)]
         [String]
-        $name = $script:StorageAccountName,
```
Comment (vikask77) [inline Scripts/AzPowerShellScript/AzHelper.psm1:201]: Can we use the same parameter name for storageaccountname, or change all commandlets to have a name parameter and let the cmdlet clarify what object it reffers to

File: Scripts/AzPowerShellScript/CreateVM.ps1 [hunk 2]
```diff
@@ -2,57 +2,59 @@
...
+	[String]$VmName,
+	[Parameter(Mandatory=$True)]
+	[String]$VMUserName,
+	[Parameter(Mandatory=$True)]
+	[String]$VMPassword,
+	[Parameter(Mandatory=$True)]
+	[ValidateSet("Windows", "Linux")]
+	[String]$OS,
 	[Parameter(Mandatory=$True)]
 	[String]$VHDPath,
```
Comment (vikask77) [inline Scripts/AzPowerShellScript/CreateVM.ps1:16]: We do not need 2 parameters to get the vhd full path.

**Review by vikask77 — APPROVED** [review]

File: Scripts/AzPowerShellScript/CreateVM.ps1 (hunk 2 above)
Comment (neerajdixit-msft) [reply Scripts/AzPowerShellScript/CreateVM.ps1:16]: the vhd file name is required for creating a managed disk (not including path)
vhd file name and vhd file path is required for azopy

File: Scripts/AzPowerShellScript/AzHelper.psm1 (hunk 1 above)
Comment (neerajdixit-msft) [reply Scripts/AzPowerShellScript/AzHelper.psm1:201]: Fixed in next patch

**Pull request merged**